pkg/opsgenie/types.go

```diff
@@ -5,13 +5,22 @@ type IntegrationsResponse struct {
 }
 
 type Integration struct {
-	ID       string `json:"id"`
-	Name     string `json:"name"`
-	Type     string `json:"type"`
-	TeamID   string `json:"teamId"`
-	TeamName string `json:"teamName"`
-	Priority string `json:"priority"`
-	APIKey   string `json:"apiKey"`
-	Endpoint string `json:"endpoint"`
-	EnvType  string `json:"envType"`
+	ID       string  `json:"id"`
+	Name     string  `json:"name"`
+	Type     string  `json:"type"`
+	TeamID   string  `json:"teamId"`
+	TeamName string  `json:"teamName"`
+	Priority string  `json:"priority"`
+	APIKey   string  `json:"apiKey"`
+	Endpoint string  `json:"endpoint"`
+	EnvType  EnvType `json:"envType"`
 }
+
+type EnvType string
+
+const (
+	EnvTypeDev     EnvType = "dev"
+	EnvTypeStaging EnvType = "staging"
+	EnvTypeProd    EnvType = "prod"
+	EnvTypeGlobal  EnvType = "null" // Intentionally a string called "null" as this is the expected result from opsgenie int manager
```

Comment:
cc @jokeyrhyme

Reply:
?

Reply:
Why not `global`?

Reply:
I am not sure why its not called "global. @jokeyrhyme Do you know why?

Reply:
@fcobb Well, it's not that things are global, it's just that they aren't explicitly environment-specific
Maybe this should actually mean Global, and maybe we should actually emit `"global"` instead of `"null"`
When we put this together, the most important thing was making sure our CloudWatch alarms (which are environment-specific) were covered, everything else was secondary
I'm wondering whether we should call this Null or something so that we can meaningfully transition to Global in future?

Reply:
@jokeyrhyme I went for Global as if its not environment specific then its essentially global.
As you own this API I will defer these decisions to you and your team. My preference is to return a specific value for these non-environment specific values rather than null.